mapper/tedge_mapper/Cargo.toml

```diff
@@ -8,11 +8,14 @@ description = "tedge_mapper is the mapper that translates thin-edge.io data mode
 
 [package.metadata.deb]
 depends = "tedge"
-maintainer-scripts = "configuration/debian/tedge_mapper"
+maintainer-scripts = "../../configuration/debian/tedge_mapper"
+assets = [
+    ["../../configuration/init/systemd/tedge-mapper-az.service", "/lib/systemd/system/tedge-mapper-az.service", "644"],
+    ["../../configuration/init/systemd/tedge-mapper-c8y.service", "/lib/systemd/system/tedge-mapper-c8y.service", "644"],
+]
```

Comment:
One critical. This package doesn't contain tedge_mapper birary! I think it's because of assets. If we use this assets, we also need to give the path to binary. https://github.com/mmstick/cargo-deb

Reply:
Good catch, since we use assets we now have to manually add all resources to be installed on the system.